tests/tests_app/frontend/test_web.py

```diff
@@ -54,17 +55,21 @@ def test_start_server_through_function(uvicorn_mock, tmpdir, monkeypatch):
     FastAPIMock.get.return_value = FastAPIGetDecoratorMock
     monkeypatch.setattr(lightning_app.frontend.web, "FastAPI", MagicMock(return_value=FastAPIMock))
 
-    lightning_app.frontend.web.start_server(serve_dir=tmpdir, host="myhost", port=1000, path="/test-flow")
-    uvicorn_mock.run.assert_called_once_with(app=ANY, host="myhost", port=1000, log_config=ANY)
-    FastAPIMock.mount.assert_called_once_with("/test-flow", ANY, name="static")
+    root_path = "/base"
```

Comment:
could you parametrize the test with `root_path=("/", "/base")` please?

Reply:
updated!